src/qt/guiutil.cpp

```diff
@@ -912,6 +914,16 @@ void LogQtInfo()
     const std::string plugin_link{"dynamic"};
 #endif
     LogPrintf("Qt %s (%s), plugin=%s (%s)\n", qVersion(), qt_link, QGuiApplication::platformName().toStdString(), plugin_link);
+    const auto static_plugins = QPluginLoader::staticPlugins();
+    if (static_plugins.empty()) {
+        LogPrintf("No static plugins.\n");
+    } else {
+        for (const QStaticPlugin& p : static_plugins) {
+            LogPrintf("Static plugin: %s\n", QJsonDocument(p.metaData()).toJson(QJsonDocument::Compact).toStdString());
```

Comment:
nit, could write "Static plugins:" just once before the loop.

Reply:
Thanks! [Updated](https://github.com/bitcoin-core/gui/pull/72#issuecomment-702326522).